lib/src/server/server.dart

```diff
@@ -93,20 +93,33 @@ class Server {
       bool v6Only: false,
       bool shared: false}) async {
     // TODO(dart-lang/grpc-dart#9): Handle HTTP/1.1 upgrade to h2c, if allowed.
+    bool unix = false;
+    dynamic _address = address;
+    int _port = port;
+    if (address != null) {
+      // `address` which start with 'unix://' prefix, will be parsed as unix socket.
+      RegExp re = RegExp(r'(^unix://)');
+      Match m = re.firstMatch(address);
+      if (m != null) {
+          unix = true;
+          _address = InternetAddress(address.substring(m.end), type: InternetAddressType.unix);
```

Comment:
If you don't use RegExp, then `m.end` can be `"unix://".length` (or 7, if you don't mind magic constants, both are constant expressions).